example/android/settings.gradle

```diff
@@ -1,35 +1,37 @@
 pluginManagement { includeBuild("../node_modules/@react-native/gradle-plugin") }
 plugins { id("com.facebook.react.settings") }
-extensions.configure(com.facebook.react.ReactSettingsExtension){ ex -> ex.autolinkLibrariesFromCommand() }
+extensions.configure(com.facebook.react.ReactSettingsExtension) { ex -> ex.autolinkLibrariesFromCommand() }
 rootProject.name = 'KlaviyoReactNativeSdkExample'
 include ':app'
 includeBuild('../node_modules/@react-native/gradle-plugin')
 
 // Substitute the jitpack SDK dependency for your local instance of the project
 // Enabling here allows you to build the example app against a local android SDK
-def localProperties = new Properties()
```

Comment:
I did try factoring out this into a re-usable gradle method but the gradle gods said no.